Signal/src/Models/TSMessageAdapaters/TSVideoAttachmentAdapter.m

```diff
@@ -310,22 +314,30 @@ - (void)performEditingAction:(SEL)action
         }
     } else if ([self isAudio]) {
         if (action == @selector(copy:)) {
-            NSData *data = [NSData dataWithContentsOfURL:self.fileURL];
-
-            NSString *pasteboardType = [MIMETypeUtil getSupportedExtensionFromAudioMIMEType:self.contentType];
-
-            if ([pasteboardType isEqualToString:@"mp3"]) {
-                [UIPasteboard.generalPasteboard setData:data forPasteboardType:(NSString *)kUTTypeMP3];
-            } else if ([pasteboardType isEqualToString:@"aiff"]) {
-                [UIPasteboard.generalPasteboard setData:data
-                                      forPasteboardType:(NSString *)kUTTypeAudioInterchangeFileFormat];
-            } else if ([pasteboardType isEqualToString:@"m4a"]) {
-                [UIPasteboard.generalPasteboard setData:data forPasteboardType:(NSString *)kUTTypeMPEG4Audio];
-            } else if ([pasteboardType isEqualToString:@"amr"]) {
-                [UIPasteboard.generalPasteboard setData:data forPasteboardType:@"org.3gpp.adaptive-multi-rate-audio"];
-            } else {
-                [UIPasteboard.generalPasteboard setData:data forPasteboardType:(NSString *)kUTTypeAudio];
+            NSString *utiType = [MIMETypeUtil utiTypeForMIMEType:_contentType];
+            if (!utiType) {
```

Comment:
Agreed. Actually, it'd be nice to have quite comprehensive "file extension -> mimetype" and "mimetype -> uti type" mappings as we do now for: `[MIMETypeUtil  genericMIMETypesToExtensionTypes:]`.  I'll file a card.